server/gw.go

```diff
@@ -0,0 +1,33 @@
+package server
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
+	"github.com/pkg/errors"
+	"google.golang.org/grpc"
+)
+
+type GRPCGateway struct {
+	// gwmux is the grpc-gateway proxy multiplexer
+	gwmux   *runtime.ServeMux
+	address string
+}
+
+func NewGateway(host string, port int) (*GRPCGateway, error) {
+	return &GRPCGateway{
+		gwmux: runtime.NewServeMux(
+			runtime.WithErrorHandler(runtime.DefaultHTTPErrorHandler),
```

Comment:
I believe this line is redundant. By default `runtime.NewServeMux` have default error handler as `runtime.DefaultErrorHandler`.

Reply:
Done.